### ISSUE TITLE
Messaging: Sales Tax notifications

### ISSUE THREAD

**Issue by joethreepwood** [PostHog/meta#154]
# Messaging

### Context
For https://github.com/PostHog/company-internal/issues/1055

We need to tell users about sales tax requirements. Charges start on 17th December.

There's no action required for users. 

### Goal
No goal. 

### Format
2x emails to all users in the impacted orgs. 

First message on Friday 8th. Reminder on Friday 15th

### Audience / Targeting
@fraserhopper @simfish85 this is one area where I'm still confused after reading the issues and threads. 

Basically: How do we plan to identify the customers which need to receive these emails, and do we need to email the NY, UK, and EU users separately or can we use one set of messages for all of them?

**Comment by fraserhopper** [comment]
So as far as identifying them, I believe we can pull these from stripe. 

I think we can do a different messaging for NY & then for Uk/Eu(these can be the same message).

We only need to message the billing contact provided to Stripe. No need to contact every user.

**Comment by joethreepwood** [comment]
OK, great. If someone can give me a list of users (don't share emails here) when they're ready, I can set this up along those lines.

**Comment by joethreepwood** [comment]
<img width="770" alt="UK-1" src="https://github.com/PostHog/meta/assets/84011561/69b78512-795e-405e-a358-9393589a6bc3">

<img width="808" alt="UK-2" src="https://github.com/PostHog/meta/assets/84011561/e5b35933-1b2a-455a-9360-dcf0e568a4f1">

Draft of the UK & EU notifications based on what I understand of the requirements here. Happy to take any feedback!

**Comment by joethreepwood** [comment]
OK, we've elsewhere decided to drop the UK email and only need to send the NYC variant to a bunch of users in the US. 

Content is unchanged, segment is created, and these emails are now scheduled to send.
